install.sh

```diff
@@ -45,21 +45,28 @@ MONDOO_BINARY="cnspec" # binary that we search for
 
 # read bash flags
 MONDOO_INSTALLER=''
+MONDOO_SERVICE=''
 MONDOO_REGISTRATION_TOKEN=''
 
 print_usage() {
   echo "usage: [-i]" >&2
   echo "  Options: " >&2
   echo "    -i <installer>:  Select a specific installer, options are:" >&2
   echo "                     macOS: brew, pkg" >&2
+  echo "    -s <service>:    Enables the cnspec service for the system." >&2
+  echo "                     options are: enable" >&2
   echo "    -t <token>:      Registration Token to authenticate with" >&2
   echo "                     Mondoo Platform" >&2
+  echo "    -u <updater>:    Enables the Mondoo auto updater for the system." >&2
+  echo "                     options are: enable" >&2
```

Comment:
Can this be used without `-s` or is it in addition to running cnspec as a service?

Reply:
-s and -u can be used independently of each other